index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>My Website</title>
+    <style>
+      /* Add CSS styling for the navigation bar and search bar */
+      #navbar {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        background-color: #333;
+        color: #fff;
+        padding: 10px;
+      }
+      #navbar a {
+        color: #fff;
+        text-decoration: none;
+        margin-right: 10px;
+      }
+      #search-container {
+        display: flex;
+        align-items: center;
+        background-color: #fff;
+        padding: 10px;
+        border-radius: 10px;
+      }
+      #search-icon {
+        margin-right: 10px;
+      }
+      #search-input {
+        border: none;
+        padding: 10px;
+        font-size: 16px;
+      }
```

Comment:
Move all these styles in the style.css file.

Reply:
Great job adding your review 👍 . I think it will be good if you give the student the reason for asking him to move the styles to the CSS file. 😊 ☺️ 

Also, kindly use polite suggestions so the student will not feel controlled e.d you can use "I think it would be better to move all these styles to CSS file because... 😉 

[OPTIONAL]I think it will be great,  if you make use of emojis and Gifs. This will make your code review more human-friendly :alien::robot: